tuf/builder.go

```diff
@@ -545,11 +557,19 @@ func (rb *repoBuilder) loadDelegation(roleName string, content []byte, minVersio
 	}
 
 	if err := signed.VerifyVersion(&(signedTargets.Signed.SignedCommon), minVersion); err != nil {
+		// don't capture in invalidRoles because the role we received is a rollback
```

Comment:
Can we add a test for this case where if the role is too low, `invalidRoles` will not contain the version that was attempted?